specification/dynamicstelemetry/resource-manager/Microsoft.DynamicsTelemetry/preview/2019-01-24/dynamicstelemetry.json

```diff
@@ -0,0 +1,139 @@
+{
+  "swagger": "2.0",
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "info": {
+    "title": "Dynamics Telemetry",
+    "version": "2019-01-24"
+  },
+  "host": "telemetry.dynamics.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/providers/Microsoft.DynamicsTelemetry/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": true,
```

Comment:
is x-nullable here used to indicate it may return a body or not? If there is no body returned sometimes, then this should have a 204 response.

Reply:
fixed per your suggestion